containers/scala/buildDockerImage.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+dockerImageName=${1:-accelerate-io/dpnt-coverage-scala}
+dockerImageVersion=${2:-0.1}
+
+echo "Building docker image for with the name ${dockerImageName}:${dockerImageVersion}"
+docker build -t ${dockerImageName}:${dockerImageVersion} ${SCRIPT_CURRENT_DIR}/.
+
+echo "Remove any dangling images from the local registry"
+imagesToRemove=$(docker images -q -f dangling=true)
+[ ! -z ${imagesToRemove} ] && docker rmi -f $imagesToRemove || true
```

Comment:
This fails if there is more than one dangling image:
```
Successfully built 896a24f03d11
Successfully tagged accelerate-io/dpnt-coverage-scala:0.1
Remove any dangling images from the local registry
./buildDockerImage.sh: line 17: [: too many arguments
```

Reply:
WIll look into this, not sure why its happening

Reply:
Done!